pkg/config/setup/system_probe.go

```diff
@@ -289,6 +289,12 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 
 	// network_config network path config
 	cfg.BindEnvAndSetDefault(join(npNS, "enabled"), false)
+	cfg.BindEnvAndSetDefault(join(npNS, "workers"), 4)
```

Comment:
My concern here is that system-probe config is being used to control things that are not run within system-probe.

Reply:
See https://github.com/DataDog/datadog-agent/pull/25044/files?show-viewed-files=true&file-filters%5B%5D=#r1601843461

Reply:
moved the configs to datadog.yaml here https://github.com/DataDog/datadog-agent/commit/2887ede563295570772260cac4630e144c953dfa and https://github.com/DataDog/datadog-agent/commit/ac08ffc1f76a2b6f95242f524506ea06640bd019